onebusaway-android/src/main/java/org/onebusaway/android/ui/NavigationDrawerFragment.java

```diff
@@ -89,7 +91,8 @@
             R.string.navdrawer_item_my_reminders,
             R.string.navdrawer_item_settings,
             R.string.navdrawer_item_help,
-            R.string.navdrawer_item_send_feedback
+            R.string.navdrawer_item_send_feedback,
+            R.string.navdrawer_item_plan_trip
```

Comment:
We should change to dynamically generating this array, and only add the "Plan a Trip" navigation drawer element for regions where `region.getOtpBaseUrl()` isn't null.